File: Models/Request.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

namespace TFCTestTask.Models;

public class Request
{
    [Key]
    public int Id { get; set; }
    public string Title { get; set;}
    public string Descriptions { get; set; }
    public string Email { get; set; }
    public DateTime DateTimeDeadline { get; set; }
    public int ApplicationId { get; set; }
    [ForeignKey(nameof(ApplicationId))]
    public Application Application { get; set;}
}

File: ClientApp/src/app/models/Request.ts
import {Application} from "./Application";

export class Request {

  Id: number;
  Title: string;
  Descriptions: string;
  Email: string;
  DateTimeDeadline: Date;
  ApplicationId: number;
  Application: Application | null;



  constructor() {
    this.Id = 1;
    this.Title = "";
    this.Descriptions = "";
    this.Email = "";
    this.DateTimeDeadline = new Date();
    this.ApplicationId = -1;
    this.Application = null;
  }

}
